x-pack/plugins/secops/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "skipLibCheck": true
```

Comment:
this can be removed when the following `react-split-pane` issue is resolved https://github.com/tomkp/react-split-pane/issues/330